app.js

```diff
@@ -182,8 +407,12 @@ app.action('coc_acknowledge', async ({ ack, body }) => {
   await sendMessage(body.channel.id, shipDesc, 10, finalTs)
   await sendMessage(body.channel.id, codeDesc, 10, finalTs)
   await sendMessage(body.channel.id, `Here are a bunch of other active channels that you may be interested in:`, 10, finalTs)
-  await sendMessage(body.channel.id, `<#C0JDWKJVA> <#C0NP503L7> <#C6LHL48G2> <#C0DCUUH7E> <#CA3UH038Q> <#C90686D0T> <#CCW6Q86UF> <#C1C3K2RQV> <#CCW8U2LBC> <#CDLBHGUQN> <#CDJV1CXC2> <#C14D3AQTT> <#CBX54ACPJ> <#CC78UKWAC> <#C8P6DHA3W> <#C010SJJH1PT> <#CDJMS683D> <#CDN99BE9L> <#CSHEL6LP5>`, 10, finalTs)
-
+  if (pronouns === "They/them/theirs" || pronouns === "She/her/hers"){
+  await sendMessage(body.channel.id, `<#C0JDWKJVA> <#C0NP503L7> <#C6LHL48G2> <#C0DCUUH7E> <#CA3UH038Q> <#C90686D0T> <#CCW6Q86UF> <#C1C3K2RQV> <#CCW8U2LBC> <#CDLBHGUQN> <#CDJV1CXC2> <#C14D3AQTT> <#CBX54ACPJ> <#CC78UKWAC> <#C8P6DHA3W> <#C010SJJH1PT> <#CDJMS683D> <#CDN99BE9L> <#CSHEL6LP5> <#CFZMXJ3FB> (an awesome channel just for girls + nb people)`, 10, finalTs)
```

Comment:
I’m worried OL will blend in a little too much here. Does it feel weird if it’s a final message in that thread?

Reply:
ya, the problem was I didn't want to auto add them so the only place left was that message:

#hack-night #hackathons #gamedev #music #support #lgbtq #apple #design #android #cats #dogs #packages #photography #jenga #food #cooking #counttoamillion #film #playdungeon

Let me think of a better way to add it in

Reply:
I've updated it, what do you think of it now?